eq-author/src/App/review/ReviewPage.js

```diff
@@ -61,23 +63,57 @@ const ReviewPage = ({ match, history }) => {
     return <Redirect to={`/q/${match.params.questionnaireId}`} />;
   }
 
+  const [rejectComment, setRejectComment] = useState("");
```

Comment:
This can be just `[comment, setComment]`

Reply:
I think the default value for useState should be {value: ""}, as that is the value the components are looking for. Or you could change `onUpdate={setRejectComment}` to `onUpdate={(e) => setRejectComment(e.value)}`.